completion/git-machete.fish

```diff
@@ -4,7 +4,7 @@
 # > if the user types git-machete, then only git-machete.fish will be loaded, but you can trick
 # > fish into still loading git.fish by adding a complete -C"git " at the top
 # - https://gitter.im/fish-shell/fish-shell?at=6176a4bacd4972068b6a0804
-complete -C "git "
+complete --do-complete="git "
 
 set -l __mcht_help_topics format hooks
 set -l __mcht_commands_long add advance anno delete-unmanaged diff discover edit file fork-point \
```

Comment:
One thing I've noticed in `fish`:

![image](https://user-images.githubusercontent.com/3383210/140104980-13b1341e-eb97-4986-8c31-99046cf39311.png)

`github` and `is-managed` don't have descriptions (probably coz they weren't included in `short_docs` until recently)... pls fill up

Reply:
(yep, that was the exaxt reason)